package.json

```diff
@@ -50,62 +50,72 @@
+        "@babel/preset-typescript": "^7.27.0",
+        "@rollup/plugin-babel": "^6.0.4",
+        "@rollup/plugin-commonjs": "^28.0.3",
+        "@rollup/plugin-json": "^6.1.0",
+        "@rollup/plugin-node-resolve": "^16.0.1",
+        "@rollup/plugin-replace": "^6.0.2",
+        "@storybook/addon-actions": "^8.6.12",
+        "@storybook/addon-essentials": "^8.6.12",
+        "@storybook/addon-links": "^8.6.12",
+        "@storybook/addon-webpack5-compiler-babel": "^3.0.6",
+        "@storybook/builder-webpack5": "^8.6.12",
+        "@storybook/html": "^8.6.12",
+        "@storybook/html-webpack5": "^8.6.12",
+        "@storybook/preset-html-webpack": "^8.6.12",
+        "@storybook/react": "^8.6.12",
+        "@storybook/react-webpack5": "^8.6.12",
+        "@storybook/test": "^8.6.12",
+        "@types/expect-puppeteer": "^5.0.6",
+        "@types/jest": "^29.5.14",
+        "@types/jest-environment-puppeteer": "^5.0.6",
+        "@types/react": "^18.3.20",
+        "@types/react-dom": "^18.3.6",
+        "@typescript-eslint/eslint-plugin": "^8.31.0",
+        "@typescript-eslint/parser": "^8.31.0",
+        "auto-changelog": "^2.5.0",
+        "babel-jest": "^29.7.0",
+        "babel-loader": "^10.0.0",
+        "babel-plugin-annotate-pure-calls": "^0.5.0",
         "copyfiles": "^2.4.1",
-        "eslint": "^8.30.0",
-        "eslint-config-prettier": "^8.5.0",
+        "eslint": "^9.25.1",
+        "eslint-config-prettier": "^10.1.2",
         "eslint-plugin-header": "^3.1.1",
-        "eslint-plugin-import": "^2.26.0",
-        "jest": "^27.5.1",
-        "jest-puppeteer": "^6.1.0",
-        "prettier": "^2.5.1",
-        "puppeteer": "^21.4.1",
-        "react": "^17.0.0",
-        "react-dom": "^17.0.0",
-        "release-it": "^15.4.1",
-        "rimraf": "^3.0.2",
-        "rollup": "^2.67.2",
-        "rollup-plugin-dts": "^4.2.0",
-        "rollup-plugin-sourcemaps": "^0.6.3",
-        "rollup-plugin-typescript2": "^0.31.2",
-        "strip-ansi": "^7.0.1",
+        "eslint-plugin-import": "^2.31.0",
+        "jest": "^29.7.0",
+        "jest-puppeteer": "^11.0.0",
+        "monosize": "^0.6.3",
+        "monosize-bundler-webpack": "0.1.6",
+        "monosize-storage-azure": "0.0.16",
+        "prettier": "^3.5.3",
+        "puppeteer": "^24.7.1",
+        "react": "^18.3.1",
+        "react-dom": "^18.3.1",
+        "release-it": "^19.0.1",
+        "rimraf": "^6.0.1",
+        "rollup": "^4.40.0",
+        "rollup-plugin-dts": "^6.2.1",
+        "rollup-plugin-typescript2": "^0.36.0",
+        "strip-ansi": "^7.1.0",
         "tree-kill": "^1.2.2",
-        "tsconfig-paths-webpack-plugin": "^3.5.2",
-        "typescript": "^4.9.4",
-        "vite": "^2.9.12"
+        "tsconfig-paths-webpack-plugin": "^4.2.0",
+        "typescript": "^5.8.3",
+        "typescript-eslint": "^8.31.0",
+        "vite": "^6.3.2"
+    },
+    "optionalDependencies": {
+        "@rollup/rollup-linux-x64-gnu": "4.40.0"
```

Comment:
This should not be needed

Reply:
This is needed because the build fails in the linux CI without it.

Reply:
You can see the error here: https://github.com/microsoft/tabster/actions/runs/14626618441/job/41039528037

Reply:
Got it